xml/OptionPanel.xml

```diff
@@ -101,6 +110,30 @@
             </Cell>
           </Row>
 
+          <!-- Option: card language -->
+          <Row class="option-text">
+            <Cell class="option-dropdowntext">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Card language</Text>
+                <Text class="description">Downloading a supported campaign or importing a deck will use this language for the cards.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-dropdown">
+              <Panel padding="0 17 13 13">
+                <Dropdown id="cardLanguage"
+                  onValueChanged="languageSelected(selectedIndex)">
+                  <Option>简体中文</Option>
```

Comment:
Can these options can be created dynamically from the language name list?

Reply:
I tried it, but after hitting my head against a wall for an hour, I give up. I assume dropdown menu works differently than other XML.